modules/search/worldcat.py

```diff
@@ -0,0 +1,84 @@
+"""
+OWASP Maryam!
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+from xml.etree import ElementTree
```

Comment:
Add it to the requirement file

Reply:
It is present in the basic python library. Do i still need to add it to the requirements ?

Reply:
Here, it is a part of the standard python library : https://docs.python.org/3/library/xml.etree.elementtree.html

Reply:
well, I check this page: https://docs.python.org/3/py-modindex.html and found it isn't in the list

Reply:
It is present on this page under the xml tab if you will expand it. Here is a screenshot https://imgur.com/a/xcxMxE1

Reply:
got it